zeebe/gateway-protocol/src/main/proto/v2/incidents.yaml

```diff
@@ -378,3 +408,55 @@ components:
       properties:
         operationReference:
           $ref: 'keys.yaml#/components/schemas/OperationReference'
+
+    IncidentStatisticsQuery:
+      type: object
+      additionalProperties: false
+      properties:
+        page:
+          allOf:
+            - $ref: 'search-models.yaml#/components/schemas/OffsetPagination'
```

Comment:
🔧 If we don't provide any additional description, we can simply this to just a `$ref`. If we want to add a dedicated description here (might actually be helpful), let's do that and keep the `allOf`.